src/FunctionWrapper.php

```diff
@@ -35,11 +32,14 @@ public function __toString()
      *
      *
      * @param callable $function
-     * @param array   $args
-     * @param bool    $return_output_buffer
+     * @param array $_args
+     * @param bool $_use_ob
      */
-    public function __construct($function, $args = [], $return_output_buffer = false)
-    {
+    public function __construct(
+        $function,
+        private $_args = [],
```

Comment:
Should we rename the properties to match the parameter names? Otherwise, this would also be a breaking change because PHP 8 has named parameter names.

Reply:
I don't get how this changed. You're right! I renamed the parameters correctly.